algorithms/vocab1.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Vocab class 1</title>
+
+    <style>
+		table {
+		    border-collapse: collapse;
+		    width: 100%;
+		}
+
+		th {
+			background-color: #dddddd;
+		}
+
+		td, th {
+		    border: 1px solid #dddddd;
+		    text-align: left;
+		    padding: 8px;
+		}
+
+		td {
+			height: 100px;
+		}
+
+	</style>
+  </head>
+  <body>
+  
+  	<h2>Vocab for class 1</h2>
+  	<table>
+	  <tr>
+	    <th>Concept</th>
+	    <th>Definition</th>
+	    <th style="width: 50%">Example</th>
+	  </tr>
+	  <tr>
+	    <td>Algorithm</td>
+	    <td>A repeatable process for determining the solution to a problem</td>
+	    <td></td>
```

Comment:
Will you be adding examples?

Reply:
The plan was people could use this to take notes if they wanted to, so I was going to leave it blank.

Reply:
Oh, interesting. Sounds good!